src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChatWidget.ts

```diff
@@ -159,7 +159,7 @@ export class TerminalChatWidget extends Disposable {
 		this._inlineChatWidget.focus();
 	}
 	focusResponse(): void {
-		const responseElement = this._inlineChatWidget.domNode.querySelector(ChatElementSelectors.ResponseEditor) || this._inlineChatWidget.domNode.querySelector(ChatElementSelectors.ResponseMessage);
+		const responseElement = this._inlineChatWidget.domNode.querySelector('.monaco-list');
```

Comment:
Resorting to querySelector is essentially using private API that could break. Can we do this via `this._inlineChatWidget.chatWidget.focusLastMessage()` or similar?